### ISSUE TITLE
Display error and add Loading with react components

### ISSUE THREAD

**Issue by LuccaGugescu** [Louvivien/tradingapp#7]
Display the error with a Toast so the user can understand the problem and add loading component

![Cattura](https://github.com/Louvivien/tradingapp/assets/61294264/00aeec14-606c-4819-9dad-7d7caf9ac854)

check for easy integration https://www.npmjs.com/package/react-toastify

**Comment by Louvivien** [comment]
Good idea. This error is fixed by the way, it was because the database was off
